Web Experiences/Blueprints/alert-bar/alertBarScroll.js

```diff
@@ -1,29 +1,36 @@
+//make experience unique
+var compiledCSS = Boxever.templating.compile(variant.assets.css)(variant);
+var styleTag = document.getElementById('style-' + variant.ref);
+if (styleTag) {
+    styleTag.innerHTML = compiledCSS;
+}
+////
+
 insertHTMLBefore("body");
 
-// Close banner
-document.querySelector("#bx_TopBanner").style.display = "none";
+// declarations
+const bxButton = document.querySelector("#bx-"+variant.ref+ ' #bx_TopBanner-button');
+const bxCloseButton = document.querySelector("#bx-"+variant.ref+ ' .bx__btn-close');
+const bxExperience = document.querySelector("#bx-"+variant.ref+ ' #bx_TopBanner');
+
+// start with hidden banner (show on scroll)
+bxExperience.style.display = "none";
+
+//accept from template when to trigger the experience
+const scrollPercentageInput = [[ Show at percentage page | enum(0,25,50,100)| 0 |{group: General}]];
 
-const scrollPercentageInput = [[Scroll Percentage | enum(0,25,50,100)||{order: 1}]]
+//Listen on scroll, if scrolled enough start experience
 window.onscroll = function() {currentScrollPercentage()};
-function currentScrollPercentage()
-{
+const currentScrollPercentage= ()=>{
     const scrollPercentage = Math.round((document.documentElement.scrollTop + document.body.scrollTop) / (document.documentElement.scrollHeight - document.documentElement.clientHeight) * 100);
-    // console.log(scrollPercentage)
     if (scrollPercentage > scrollPercentageInput){
-        document.querySelector("#bx_TopBanner").style.display = "block";
+        bxExperience.style.display = "block";
         document.body.classList.add("show-TopBanner");
     }
 }
```

Comment:
We should remove the scroll triggering from the template. Those who want it can add it in Page Targeting

Reply:
Done